rust/libnewsboat-ffi/build.rs

```diff
@@ -1,6 +1,6 @@
 fn add_cxxbridge(module: &str) {
     cxx_build::bridge(format!("src/{module}.rs"))
-        .flag("-std=c++11")
+        .std("c++14")
```

Comment:
I completely forgot that we specify the version here! Please add cross-comments here and in Makefile so we don't forget the next time we bump the flag.